src/containerapp/azext_containerapp/_clients.py

```diff
@@ -492,3 +492,106 @@ def list(cls, cmd, resource_group_name, env_name, formatter=lambda x: x):
                 env_list.append(formatted)
 
         return env_list
+
+
+class BuilderClient():
+    api_version = PREVIEW_API_VERSION
+
+    @classmethod
+    def list(cls, cmd, resource_group_name):
```

Comment:
one more question: as builder/build are azure resources, `up` command will help create the resource implicitly when --source is specified, but when container app is deleted, the builder resource will still be there, right?

so how can customer delete it as I do not see delete operation for your client?

Reply:
As of today, there is no way in the CLI to delete the builder. This is because the builder is, as of today, a different concept from the container apps environment but that in the coming months, it will be a part of it and as such its lifecycle will be handled with the container apps environment. Please note that the builder does not occur costs for the customer.